hubtools/src/bootleby.rs

```diff
@@ -72,10 +72,7 @@ fn add_image_header(path: PathBuf) -> Result<Vec<u8>, Error> {
     let offset = header_offset(&elf)?;
     drop(elf);
 
-    let header = header::ImageHeader {
-        magic: 0x64_CE_D6_CA,
-        total_image_len: len as u32,
-    };
```

Comment:
This was an intentional choice when this was added to only add the magic and length fields to save space. It feels like a step in the wrong direction to add the full header back.